src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -343,7 +434,7 @@ class FastSync(
         val evm = account.map(_.codeHash)
         val storage = account.map(_.storageRoot)
 
-        blockchain.saveNode(ByteString(n.hash), n.toBytes, initialSyncState.targetBlock.number)
+        blockchain.saveNode(ByteString(n.hash), n.toBytes, syncState.targetBlock.number)
```

Comment:
hehe this one could lead to tricky errors if you forgot to update it :smile:

Reply:
heh, this one has bitten me at the beginning of work at this pr :P